src/components/input-field/examples/input-field-url.tsx

```diff
@@ -0,0 +1,78 @@
+import { Component, h, State, Watch } from '@stencil/core';
```

Comment:
These three examples are quite similar. What do you think of combining them into a single one? :thinking:

Reply:
Yeah, I've considered it.

I think combining them makes the live example better, but it also makes the code-side of the example worse, as it becomes more complex. I'll ask the add-ons team what they think.

Reply:
Katja was about as undecided as I am… 🤷‍♂️

Reply:
Haha ok, I will let you decide what you think is best. My argument for combining them is that there will be less code to maintain and I don't think the example would be _that_ much more complex :slightly_smiling_face:

Reply:
I've pushed a fixup where all three types still have their own fields, but they've been combined into a single example. Not sure that's any better. Showing only one field but with a select for the type might be better, but I'm a little too tired to take on doing that right now. So try this one out and see what you think.

Reply:
I think this looks good! :+1: